### DISCUSSION TITLE
How to invoke export from my code?

### DISCUSSION THREAD

**Discussion by jhgarrison** [joyfullservice/msaccess-vcs-addin#572] [Q&A] [answered]
Previously there was a function `LoadAddInAndRunExport()`, but in v4 that fails in `modImportExport` at `Log.ProgressBar.Reset` with `Run-time error 91: Object variable or Witn block variable not set`, obviously because the export UI isn't visible and output is going to the immediate window.

Is there a way to invoke an export from my code?

**Comment by joyfullservice** [comment, marked as answer]
I might have to check further into that error...

In the mean time, you may be able to update the LoadAddInAndRunExport function to call `Application.Run "MSAccessVCS.HandleRibbonCommand", "btnExport"`

That should run the export the same way it runs if you click the button in the ribbon.

**Reply by jhgarrison** [reply]
That works, thanks

**Reply by hecon5** [reply]
@joyfullservice, that bug is one I encountered (and tried to fix) with PR #558; though you'll need to address the PR #557 first (it has dependancies).
